tests/migrations/test_commands.py

```diff
@@ -1152,6 +1152,14 @@ def test_makemigrations_migration_path_output(self):
             call_command("makemigrations", "migrations", stdout=out)
             self.assertIn(os.path.join(migration_dir, '0001_initial.py'), out.getvalue())
 
+        out = six.StringIO()
+        with self.temporary_migration_module() as migration_dir:
+            # relpath can raise a ValueError when it's impossible to
```

Comment:
Maybe this would be better as a separate test method with these details as the docstring?